reaktive/src/commonTest/kotlin/com/badoo/reaktive/utils/arrayqueue/ArrayQueueTest.kt

```diff
@@ -10,24 +11,28 @@ class ArrayQueueTest {
     private val queue = ArrayQueue<String?>()
 
     @Test
+    @JsName("initialSizeIsZero")
```

Comment:
we'll have the same problem with android tests. maybe let's use camel case naming by default?

Reply:
Good idea